src/docs/development/ui/animations/tutorial.md

```diff
@@ -714,50 +705,44 @@ to opaque.
 
 Each tween manages an aspect of the animation. For example:
 
-<!-- skip -->
-{% prettify dart %}
-final AnimationController controller =
-    AnimationController(duration: const Duration(milliseconds: 2000), vsync: this);
-final Animation<double> sizeAnimation =
-    Tween(begin: 0.0, end: 300.0).animate(controller);
-final Animation<double> opacityAnimation =
-    Tween(begin: 0.1, end: 1.0).animate(controller);
-{% endprettify %}
+<?code-excerpt "animate5/lib/main.dart (tweens)" plaster="none"?>
+```dart
+controller =
+    AnimationController(duration: const Duration(seconds: 2), vsync: this);
+sizeAnimation = Tween<double>(begin: 0, end: 300).animate(controller);
+opacityAnimation = Tween<double>(begin: 0.1, end: 1).animate(controller);
+```
 
 You can get the size with `sizeAnimation.value` and the opacity
 with `opacityAnimation.value`, but the constructor for `AnimatedWidget`
 only takes a single `Animation` object. To solve this problem,
-the example creates its own Tween objects and explicitly calculates the
+the example creates its own `Tween` objects and explicitly calculates the
 values.
 
-`LogoApp` widget was changed to encapsulate its own `Tween` objects.
```

Comment:
Here is the part of the prose where it mismatches with the code. It's not the `LogoApp` that's impacted, its the `AnimatedLogo` class.